scipy/spatial/kdtree.py

```diff
@@ -222,6 +222,9 @@ class KDTree(cKDTree):
         is the boxsize along i-th dimension. The input data shall be wrapped
         into :math:`[0, L_i)`. A ValueError is raised if any of the data is
         outside of this bound.
+    workers : int, optional
+        Number of workers to use for parallel processing. If -1 is given all
```

Comment:
It should say this is for parallel build, so the users do not assume the queries will be parallel as well.

Reply:
Changed to "[...] for parallel build."